packages/components/src/Radio/Radio/Radio.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { InputHTMLAttributes } from "react"
+import classnames from "classnames"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./Radio.module.scss"
+
+export type RadioProps = OverrideClassName<
+  Omit<InputHTMLAttributes<HTMLInputElement>, "type" | "checked">
+> & {
+  id: string
```

Comment:
Why is `id` required when we do nothing with it? Noticed this when I saw RadioField made it optional.
Maybe just make it optional here (it won't break anything), and everyone can be happy.

Reply:
Isn't that because we have a generate an id in the RadioField if one is not provided, whereas a Radio should always and id that the a label will point to?

Reply:
Yeah `Radio` itself would always need an `id` to link it to a `label`

Reply:
I suppose so. I wonder if this is the right place to enforce that though (maybe it is).
Maybe just add a comment above to explain why then (which should flow onto the docs).